printer_mods/TheOtherGuy110/Back_Panel_Alternative_Mod/README.md

```diff
@@ -0,0 +1,16 @@
+This PR contains 1 printer mod (3 STL's) for the VORON 0
+
+- One file for the left half
+- One file for the right half
+- And one file for the right half that includes mounting and a cutout for a 3010 fan if you want to add some form of active exhaust (I plan to work on this in the future and will add findings or updates as necessary)
+
+I printed the first and current revision of this mod on my own V0. They work and fit as intended on my machine but I may make modifications to them if others have issue.
+
+I created this mod so that I could fully enclose the print chamber of the V0 without suffocating the RPi and stepper motors in the hot air. This mod is designed to be a **replacement** for the stock back panel, and is intended to **work in tandem with** [Weaslus' Electronics Bay Seperator](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Weaslus/Electronics-Bay-Seperator)
```

Comment:
Please make this link relative

    [Weaslus' Electronics Bay Seperator](../../Weaslus/Electronics-Bay-Seperator)